### ISSUE TITLE
Other IoU MOS values in the evaluation

### ISSUE THREAD

**Issue by John155** [PRBonn/4DMOS#16]
I tried to replicate your results with the pretrained models. I followed all the instructions from the readme.  After setting everything up in docker, I use the following command to evaluate the pretraint model with 10 scan:

```
python scripts/predict_confidences.py -w xxx/xxx/10_scans.ckpt -dt 0.1 -seq 8
```
```
python scripts/confidences_to_labels.py -p predictions/ID/POSES -seq 8 -prior 0.25 -dt 0.1 -s bayes
```
**IoU MOS evaluation**
```
cd semantic-kitti-api
 ./evaluate_mos.py --dataset /path/to/kitti/dataset/ --predictions /path/to/method_predictions --split train/valid/test # depending of desired split to evaluate
```
**My result is:**
57.0 this is less than you got in the paper of 77.2. 
I don't understand why?

**Comment by benemer** [comment]
Hey @John155,

Please check out #5, this question was answered there.

TL;DR: For validation (and only validation) we use the SuMa poses without loop closing. There is some local drift in sequence 08 that reduces the IoU a lot and therefore makes it hard to compare the different variations of our model.

Best regards
Benedikt
